### ISSUE TITLE
Image Preloader with URLs

### ISSUE THREAD

**Issue by Hackzzila** [tflearn/tflearn#866]
The docs say that you can use URLs in the image preloader `loads images on the fly (from disk or url)` but looking through the code and testing it myself it looks like it only supports files. It would be great if it could support remote images.

**Comment by aymericdamien** [comment]
Yes, unfortunately it can only load image from disks now. I will give it a shot . Or if anyone want to send a PR, here is the code that needs to be changed: https://github.com/tflearn/tflearn/blob/master/tflearn/data_utils.py#L539 (check if URL or disk path).
